src/code42cli/cmds/search/cursor_store.py

```diff
@@ -30,15 +30,20 @@ def get(self, cursor_name):
         try:
             location = path.join(self._dir_path, cursor_name)
             with open(location) as checkpoint:
-                return float(checkpoint.read())
+                try:
+                    return float(checkpoint.read())
+                except ValueError:
```

Comment:
I based the `alert` checkpoints off the existing `audit-log` checkpoints, and found that the audit logs threw an error that just directed you to the log when it ran out of checkpoints (when trying to convert an empty string to a float).

I added this catch to at least send a more specific message for now but I'm not sure if there's a different solution that would be better - For instance, checking if the checkpoint is empty and then just returning an empty string instead of trying to do the type conversion.

Reply:
I'm not quite sure what you mean by `ran out of checkpoints`, since there should only be one saved from the last run (that is now the starting point for the next run). Could an empty string have been accidentally saved while testing the code? 

Although it isn't a bad idea to raise a helpful error message here if we do happen to get bad data into a checkpoint file. Maybe something like `Unable to parse checkpoint from {checkpoint_file_path}, expected a unix-epoch timestamp, got {bad_data}.` That way if a user was looking at the checkpoint file and accidentally added a character or something, they'd have the info they needed to fix it (or delete the file to reset).

Reply:
Thats a more helpful message, I'll adjust that! 

To your first point, maybe I should've said "ran out of events," as in there is no stored checkpoint because there are no more events - if that makes sense? Like if I run an `audit-log` search and make subsequent runs until all the events are returned, the checkpoint file is empty.  This throws an error when converting to a float.

Reply:
The checkpoint file should always have _something_ in it at the end of each CLI run (unless there aren't any events to be found on the very first run), as we're mainly checkpointing to be able to pick up where we left off between CLI invocations. 

Example: 

If I were an analyst wanting to just quickly check over the audit logs each day (but didn't want to have to see events I've already seen), I could run `code42 audit-log search -b 1d --use-checkpoint daily` today, look over the events, and the timestamp of the last event in the list would get saved to the `daily` checkpoint file. 

Then each subsequent day I could re-run `code42 audit-log search --use-checkpoint daily` and I'd only get events I haven't yet seen, cause the checkpoint is always updating at the end of each run with the latest seen timestamp (for file events, it'll now be the latest seen eventId).

Reply:
> unless there aren't any events to be found on the very first run

This could definitely be possible, especially depending on what the query was. Would we try to save an empty string in this scenario?

Reply:
Looks like it doesn't create the checkpoint file if no results are found, but if the command gets cancelled before the checkpoint is written we can end up with a blank checkpoint file, which then causes the `ValueError: could not convert string to float: ''`

So maybe we should handle the empty-file condition and just return None, but if there's _something_ in the checkpoint file but we can't parse it raise the `Unable to parse` message from above.